apps/gitness/src/styles.css

```diff
@@ -0,0 +1 @@
+@import '@harnessio/playground/src/styles.css';
```

Comment:
i think we need canary styles as well

Reply:
Playground internally includes `canary` styles as well, will it still need it?